style.css

```diff
@@ -100,6 +102,282 @@ ul li {
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.multi-project .project-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.multi-project .project-tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 6px;
+  width: 80%;
+  padding-top: 5px;
+}
+
+.multi-project .project-tag p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 11px;
+  line-height: 20px;
+  color: #091e42;
+  background: #fff;
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+}
+
+.multi-project .project-view {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  margin-top: 8px;
+  width: 134px;
+  height: 48px;
+  background: #ff6b00;
+}
+
+.project-view a {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  width: 200px;
+  height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  cursor: pointer;
+}
+
+/* all projects multi section end */
+
+/* single projects start */
+
+.all-projects .single-project {
+  width: 100%;
+  height: 738px;
+}
+
+.all-projects .single-project .project-image {
+  width: 100%;
+  height: 85%;
+  background: url('./images/project_image.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 140px;
+  gap: 15px;
+  overflow: hidden;
+}
+
+.all-projects .single-project .project-image .single-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 35px;
+  color: #fff;
+  line-height: 30px;
+}
+
+.all-projects .single-project .project-image .single-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #fff;
+  line-height: 40px;
+}
+
+.all-projects .single-project .project-image .single-language {
+  display: flex;
+  gap: 12px;
+  flex-direction: row;
+}
+
+.all-projects .single-project .project-image .single-language p {
+  padding: 10px 12px;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  text-align: center;
+}
+
+.single-project .view-project {
+  width: 100%;
+  height: 15%;
+  background-color: #ff6b00;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.single-project .view-project #view-project-id {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly adjust the margin , font size of the cards and size of the image for project list and also the size of the see project button Just Like the Figma Design.

The Figma Design Card look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/213771170-091c959c-697c-490c-a803-f01636324f39.PNG)

Your Project Cards Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/213771556-82abd06a-be9c-40d0-816b-73b451d5047a.PNG)

![Capture](https://user-images.githubusercontent.com/93598189/213771876-6bbfdb92-d4a2-4160-b574-c882253ab2bf.PNG)